tests/foreman/cli/test_host.py

```diff
@@ -647,6 +667,30 @@ def test_negative_create_with_unpublished_cv(self):
                 'organization-id': self.new_org['id'],
             })
 
+    @run_only_on('sat')
+    @tier3
+    @upgrade
+    def test_positive_katello_and_openscap_loaded(self):
+        """Verify that command line arguments from both Katello
+        and foreman_openscap plugins are loaded and available
+        at the same time
+
+        :id: 5b5db1d4-50f9-45a0-bb92-4571fc8d729b
+
+        :expectedresults: Command line arguments from both Katello
+            and foreman_openscap are available in help message
+            (note: help is generated dynamically based on apipie cache)
+
+        :CaseImportance: Medium
+
+        :BZ: 1671148
+        """
+        help_output = Host.execute('host update --help')
```

Comment:
a couple of questions: 
1. I'm ok adding this test here. But an adding as a unit test would be good ? what you think. 
2. ```Host.execute('host update --help')``` is any way put this into CLI command builder, if not i'm ok with this.

Reply:
As discussed on IRC, this would fit BATS suite rather well. This is very good idea. I'm only not sure if we can put it in upstream, as it requires two plugins to be installed at the same time. I'll look into this and get back to you, hopefully with new commit that removes this case. Adding do not merge flag for now.

Reply:
Worth keeping here for downstream